src/instructlab/eval/mtbench.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Local
+from .evaluator import Evaluator
+
+
+class MT_Bench_Evaluator(Evaluator):
+    """
+    Child class of an Evaluator for Multi-turn Benchmark (MT-Bench)
+
+    Attributes
+        server_url  vLLM server endpoint
+    """
+
+    def __init__(self, model_path, server_url: str) -> None:
+        super().__init__(model_path)
+        self.server_url = server_url
+
+    def run(self) -> tuple:
```

Comment:
Ali needed to change this to gen_answers and judge_answers.

Reply:
Can we merge this and iterate on it? I'd rather just have the code change subsequently then try to align with something currently in-flight.

Reply:
The other option is to include these commits with the impl commits + any needed modifications.

Reply:
This is just a skeleton PR - not meant to be all of the implementation as well. I think it makes much more sense to merge this now - it also allows @alinaryan to start from the same point to start the MMLU implementation.

Reply:
Good point about blocking mmlu.  I'm good to merge as is.